cherrypy/lib/static.py

```diff
@@ -28,6 +29,22 @@ def _setup_mimetypes():
 _setup_mimetypes()
 
 
+def _utf8_content_disposition(disposition, file_name):
```

Comment:
I think the function name can be improved here:
1) functions (unlike variables) do some actions which should be reflected in their name with a verb, the current name looks rather like a property.
2) "utf8" seems to be an implementation detail from the caller's PoV so I'd drop it.


```suggestion
def _make_content_disposition(disposition, file_name):
```